namer/namer.cc

```diff
@@ -1256,21 +1260,22 @@ class SymbolDefiner {
 
         auto scope = ensureScopeIsClass(ctx, squashNames(ctx, staticField.scopeClass, contextClass(ctx, ctx.owner)),
                                         staticField.name, staticField.asgnLoc);
+        auto name = staticField.name;
         auto sym = ctx.state.lookupStaticFieldSymbol(scope, staticField.name);
         auto currSym = ctx.state.lookupSymbol(scope, staticField.name);
```

Comment:
Maybe use `name` here to make things clearer?

Reply:
PR to come